01-instructor_guide.Rmd

```diff
@@ -1 +1,38 @@
-# (PART\*) Instructor Guide {-}
+# Instructor Guide {-}
+
+The primary goal of this exercise is to get students comfortable interacting with the R console.  Students will complete several `swirl` lessons introducing basic R concepts, and then apply their new skills by conducting a brief exploration of a dataframe.
+
+## Teaching Recommendations {-}
+
+This exercise is designed to be used either
+
+1. As a stand-alone "taste of R", to introduce students to the field of data science, give them an idea of what a data scientist's day-to-day work looks like, and help students see it as something they could do.
+1. As an entry point for a more in-depth course or workshop.  In this case we recommend following up on important topics using additional resources (such as [R for Data Science](https://r4ds.had.co.nz/)).  The `swirl` lessons are great for letting students work through the basics in a "real" R environment where they get immediate feedback, but they are hard for students to refer back to later.  We find `swirl` most useful for giving students a first look at the big ideas, along with practice typing commands into the console.  This sets them up well for a more in-depth treatment over the subsequent course/workshop sessions.
+
+The `swirl` lessons touch on several additional R commands and concepts beyond those mentioned in the learning objectives and provide a useful starting point for going deeper if you desire.  Additional `swirl` lessons can be found at the [swirl Course Network](http://swirlstats.com/scn/).
+
+## Lesson Plan  {-}
```

Comment:
Love the table!  Worth using the same titles in the Activity column as is used in the Table of Contents?

Reply:
Good catch.  It's closer now.  I do want to list out the optional challenge exercise as it's own thing in the Instructor timeline, but in the TOC it's underExplore `us_contagious_diseases`, so it's not a perfect match.